pkg/descheduler/pod/pods.go

```diff
@@ -124,3 +131,21 @@ func IsPodWithLocalStorage(pod *v1.Pod) bool {
 func OwnerRef(pod *v1.Pod) []metav1.OwnerReference {
 	return pod.ObjectMeta.GetOwnerReferences()
 }
+
+// IsPDBAllowsEviction returns true when pod eviction is allowed by PDB
+func IsPDBAllowsEviction(ctx context.Context, client clientset.Interface, pod *v1.Pod) (bool, error) {
+	PDBList, err := client.PolicyV1beta1().PodDisruptionBudgets(pod.Namespace).List(ctx, metav1.ListOptions{})
```

Comment:
Better to use `PodDisruptionBudgetInformer` instead of clientSet to get better performance. `GetPodPodDisruptionBudgets` method is perfect for this situation

Reply:
Yep, I'm preffer using informer instead, but considering losts get/list actions use clientset directly so I think maybe descheduler DON'T intend to use too much informers.

Never mind, modify this later